docs/src/reference/expressions.md

```diff
@@ -0,0 +1,299 @@
+# Expressions
+
+Expressions can be used to access variables and calculate values dynamically.
+
+The following attributes of BPMN elements **require** an expression:
+* Sequence Flow on an Exclusive Gateway: [condition](/bpmn-workflows/exclusive-gateways/exclusive-gateways.html#conditions)
+* Message Catch Event / Receive Task: [correlation key](/bpmn-workflows/message-events/message-events.html#messages)
+* Multi-Instance Activity: [input collection](/bpmn-workflows/multi-instance/multi-instance.html#defining-the-collection-to-iterate-over), [output element](/bpmn-workflows/multi-instance/multi-instance.html#collecting-the-output)
+* Input/Output Variable Mappings: [source](/reference/variables.html#inputoutput-variable-mappings)
+
+Additionally, the following attributes of BPMN elements can define an expression **optionally** instead of a static value:
+* Timer Catch Event: [timer definition](/bpmn-workflows/timer-events/timer-events.html#timers)
+* Service Task: [job type](/bpmn-workflows/service-tasks/service-tasks.html#task-definition), [job retries](/bpmn-workflows/service-tasks/service-tasks.html#task-definition)
+* Call Activity: [process id](/bpmn-workflows/call-activities/call-activities.html#defining-the-called-workflow)
+
+## Expressions vs. Static Values
+
+Some attributes of BPMN elements, like the timer definition of a timer catch event, can be defined either as a static value (e.g. `PT2H`) or as an expression (e.g. `= remaingTime`).
+
+The value is identified as an expression if it starts with an **equal sign** `=` (i.e. the expression prefix). The text behind the equal sign is the actual expression. For example, `= remaingTime` defines the expression `remaingTime` that access a variable with the name `remaingTime`.
+
+If the value doesn't have the prefix then it is used as static value. A static value is used either as a string (e.g. job type) or as a number (e.g. job retries). A string value must not be enclosed in quotes.
+
+Note that an expression can also define a static value by using literals (e.g. `"foo"`, `21`, `true`, `[1,2,3]`, `{x: 22}`, etc.).
+
+## The Expression Language
+
+An expression is written in **FEEL** (Friendly Enough Expression Language). FEEL is part of the OMG's DMN (Decision Model and Notation) specification. It is designed to have the following properties:
```

Comment:
Can we refer here to an external link of the DMN and FEEL spec?

Reply:
There is a reference to the DMN spec on the bottom. It links to the PDF.